### DISCUSSION TITLE
Page view mode as default

### DISCUSSION THREAD

**Discussion by FajnB** [koreader/koreader#8422] [Q&A] [answered]
Hi, I just wonder, how do I set page view mode on default for example on .cbz comics or picture format book, like pdf etc.

**Comment by Frenzie** [comment, marked as answer]
You can long press on the setting to set default.

**Reply by FajnB** [reply]
Oh, that is a nice touch, little but powerful feature.
Is it mentioned in the documentation? Maybe I'm blind, but I'm almost positive, that I did not see that there.

**Reply by poire-z** [reply]
Phew, we didn't miss mentionning this in the quick start guide (and it must be in the  User guide PDF):
https://github.com/koreader/koreader/blob/7baeebc2d41244020cc7d8cb5249a5a64d0e401c/frontend/ui/quickstart.lua#L74-L80

**Reply by FajnB** [reply]
Then it is truly my bad. I'm sorry. But still I am grateful that, even it is mentioned in the documents and I did oversee it, you answered.
This is signal for me, that you really care for this project. 😉
Keep it up, so far this reader is way superior to my Pocketbooks native reader, yes it does eat more power, but the feature set is just worth it.
